src/test/common/CMakeLists.txt

```diff
@@ -88,6 +88,13 @@ add_executable(unittest_util
 add_ceph_unittest(unittest_util ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/unittest_util)
 target_link_libraries(unittest_util global ${BLKID_LIBRARIES})
 
+# unittest_random
```

Comment:
I'm happy to (done), but had added it because that seems to be the general pattern in the existing file.